documentation/commands.md

```diff
@@ -197,22 +233,8 @@ print out above the command bar.
 
 _Syntax -_ `AAL123 sr`
 
-### ~~Fix~~
-~~_Aliases -_ `f` / `fix` / `track`~~
-
-~~_Syntax -_ `AAL123 f [fixname]`~~
-*_This command has been deprecated_*
-
-### Proceed Direct
-_Aliases -_ `direct` / `pd` / `dct`
-
-_Information -_ This command instructs the aircraft to go direct to a
-navigational fix, taking a shortcut. For example, if an aircraft is flying
-to fixes [A, B, C], issuing the command "pd B" will cause the aircraft
-to go to B, then C. After flying past the last fix, the aircraft will
-continue flying straight.
-
-_Syntax -_ `AAL123 pd [fixname]`
+## Aircraft Control commands
```

Comment:
I suggest `Basic Control Instructions` because it's an industry term that means the three most basic ways to control how an aircraft flies (its speed, its direction, and its altitude). So let's change this to `## Basic Control Instructions`, including only `altitude`, `heading`, and `speed`, and make it the new first section of the commands.